test/integration/graphql_test.lua

```diff
@@ -665,7 +743,8 @@ g.test_custom_type_scalar_variables = function()
         ).data.test_custom_type_scalar_list, 'echo'
     )
 
-    t.assert_error_msg_equals('Could not coerce "inputObject" to "CustomString"', function()
+    t.assert_error_msg_equals('Could not coerce value type "inputObject" with value "nil" ' ..
```

Comment:
Actually input value is not a string.

Reply:
Yes. This test with similar error message was before this patch

Reply:
But it didn't show a "nil" value. Therefore it couldn't confuse user.

Reply:
Oh, yes, of course, I'll fix it